kedro-training/docs/01_prerequisites.md

```diff
@@ -1,3 +1,53 @@
+# Training pre-requisites
+This Kedro training materials assume the following pre-requisites for your technical understanding.
+
+We assume understanding of these technologies and concepts during the training. In order to maximise your learning experience from the Kedro training, please review them before going through the training (there are external resource links for you to get a refresher on each topic).
+
+- Intermediate Python knowledge
+  - [Basic Python](https://developers.google.com/edu/python)
+    - Functions, loops, conditional statements
+    - Common data structures: lists, dictionaries, tuples
+    - [IO operation](https://realpython.com/working-with-files-in-python/) (reading and writing data)
+  - [Importing Python packages using `pip`](https://www.w3schools.com/python/python_pip.asp) and [Dependency management with `requirements.txt`](https://note.nkmk.me/en/python-pip-install-requirements/)
+  - [Working on multiple Python files in nested directory structures](https://drivendata.github.io/cookiecutter-data-science/)
```

Comment:
I feel like this requirement is a little too specific 😅

Reply:
Agree, removed :)